CMakeLists.txt

```diff
@@ -324,20 +332,39 @@ endif()
 # ==================================================================================================
 # Functions
 # ==================================================================================================
-function(list_licenses OUTPUT MODULES)
-    set(CONTENT)
-    set(_MODULES ${MODULES} ${ARGN})
-    foreach(module ${_MODULES})
-        set(license_path "../../third_party/${module}/LICENSE")
-        get_filename_component(fullname "${license_path}" ABSOLUTE)
-        string(APPEND CONTENT "License and copyrights for ${module}:\n\n")
-        file(READ ${license_path} license)
-        string(APPEND CONTENT ${license})
-        string(APPEND CONTENT "\n\n")
-    endforeach()
-    configure_file(${FILAMENT}/build/licenses.inc.in ${OUTPUT})
-endfunction(list_licenses)
-
+if (MSVC)
+    ## The MSVC compiler has a limitation on litteral string length which is reached when all the 
+    ## licenses are concanated together into a large string... so split them into multiple strings.
+    function(list_licenses OUTPUT MODULES)
+        file(WRITE ${OUTPUT} " R\"FILAMENT__(")
+        set(_MODULES ${MODULES} ${ARGN})
+        foreach(module ${_MODULES})
+            set(license_path "../../third_party/${module}/LICENSE")
+            get_filename_component(fullname "${license_path}" ABSOLUTE)
+            file(APPEND ${OUTPUT} ")FILAMENT__\", \nR\"FILAMENT__(License and copyrights for ${module}:\n\n)FILAMENT__\", \nR\"FILAMENT__(")
+            file(READ ${license_path} license_long)
+            string(REPLACE "\n" ")FILAMENT__\", \nR\"FILAMENT__(" license ${license_long})
+            file(APPEND ${OUTPUT} ${license})
+            file(APPEND ${OUTPUT} ")FILAMENT__\", \nR\"FILAMENT__(\n\n)FILAMENT__\", \"")
+        endforeach()
+        file(APPEND ${OUTPUT} "\"")
+        #file(APPEND ${OUTPUT} ")FILAMENT__\"\n")
```

Comment:
please delete

Reply:
We need to merge the two so we don't lose the changes from @prideout [here](https://github.com/google/filament/commit/394db90ec97ed8f9315689529679650935a7a407#diff-af3b638bc2a3e6c650974192a53c7291)

Reply:
I meant just the commented-out line